packages/gatsby-transformer-asciidoc/src/gatsby-node.js

```diff
@@ -23,7 +23,7 @@ async function onCreateNode(
   },
   pluginOptions
 ) {
-  if (!unstable_shouldOnCreateNode({ node }, pluginOptions)) {
+  if (!shouldOnCreateNode({ node }, pluginOptions)) {
     return
   }
```

Comment:
We should remove this check in `onCreateNode` as it will run check twice unnecesairly. This made sense when we fresh introduced the `unstable_shouldOnCreateNode`, as plugins starting using that, but core version might not have supported it yet, but we should drop it now